src/inc/volatile.h

```diff
@@ -120,7 +120,7 @@ T VolatileLoad(T const * pt)
     STATIC_CONTRACT_SUPPORTS_DAC_HOST_ONLY;
 
 #ifndef DACCESS_COMPILE
-#if defined(_ARM64_) && defined(__GNUC__)
+#if defined(_ARM64_) && defined(__GNUC__) && !(defined(__BIONIC__))
```

Comment:
@janvorli Removing the `volatile` did not fix the issue.  The compiler points to `&val` being the issue

```
__atomic_load((T const *)pt, &val, __ATOMIC_ACQUIRE);
                                                    ^~~~
```